### PR TITLE
Update references in preparation for ID1

### PR DESCRIPTION
**Pull request by selfissued**
Updates references and the uses of them

### CODE REVIEW

**Review by bc-pi — APPROVED** [review]

**Review by Sakurann — APPROVED** [review]
looks fine

**Review by peppelinux — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Access to this API is authorized using OAuth 2.0 [@!RFC6749], i.e., the Wallet u
 
 ## Requirements Notation and Conventions
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 [@!RFC2119].
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all capitals, as shown here.
```
Comment (tplooker) [inline openid-4-verifiable-credential-issuance-1_0.md:55]:
```suggestion
The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [@!RFC2119] and [@!RFC8174] when, and only when, they appear in all capitals, as shown here.
```

**Review by tplooker — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md:55]: https://www.rfc-editor.org/rfc/rfc8174.html#section-2 contains this text exactly - without the "and".

Are you willing to withdraw this suggestion, @tplooker?

Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:55]: @tplooker already approved, so I assume, yes.

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -2012,7 +1991,7 @@ This section defines Credential Format Profiles for a few of the commonly used C
 
 ## W3C Verifiable Credentials
 
-Sections 6.1 and 6.2 of [@VC_DATA] define how Verifiable Credentials MAY or MAY NOT use JSON-LD. As acknowledged in Section 4.1 of [@VC_DATA], implementations can behave differently regarding processing of the `@context` property whether JSON-LD is used or not.
+Sections 6.1 and 6.2 of [@VC_DATA] define how Verifiable Credentials MAY or MAY NOT use JSON-LD [@!JSON-LD]. As acknowledged in Section 4.1 of [@VC_DATA], implementations can behave differently regarding processing of the `@context` property whether JSON-LD is used or not.
```
Comment (jogu) [inline openid-4-verifiable-credential-issuance-1_0.md:1994]: If VC_DATA isn't a normative reference I'm not convinced that JSON-LD is a normative reference?

As JSON-LD is only mentioned in the context of being a normative dependency of VC_DATA, either they're both normative or they're both not?

Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md:1994]: Agreed.  I'll make it non-normative.

Comment (selfissued) [inline openid-4-verifiable-credential-issuance-1_0.md:1994]:
```suggestion
Sections 6.1 and 6.2 of [@VC_DATA] define how Verifiable Credentials MAY or MAY NOT use JSON-LD [@JSON-LD]. As acknowledged in Section 4.1 of [@VC_DATA], implementations can behave differently regarding processing of the `@context` property whether JSON-LD is used or not.
```

**Review by awoie — APPROVED** [review]

**Review by jogu — APPROVED** [review]

**Pull request merged**